<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KoConfigAuthorPage</class>
 <widget class="QWidget" name="KoConfigAuthorPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>695</width>
    <height>342</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <property name="spacing">
      <number>12</number>
     </property>
     <item row="5" column="2">
      <widget class="QLabel" name="lblPhoneWork">
       <property name="text">
        <string>Wor&amp;k Number:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="buddy">
        <cstring>lePhoneWork</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lblTitleDesc">
       <property name="text">
        <string>Title:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="buddy">
        <cstring>leTitle</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="leTitle"/>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="lePhoneHome"/>
     </item>
     <item row="1" column="3">
      <widget class="QLineEdit" name="lePosition"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="lblPostalDesc">
       <property name="text">
        <string>Postal code:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="buddy">
        <cstring>lePostal</cstring>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QLineEdit" name="leCountry"/>
     </item>
     <item row="2" column="3">
      <widget class="QLineEdit" name="leEmail"/>
     </item>
     <item row="4" column="2">
      <widget class="QLabel" name="lblCountryDesc">
       <property name="text">
        <string>Country:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="buddy">
        <cstring>leCountry</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="fullNameLabel">
       <property name="text">
        <string>Name:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="leFullName">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="lePostal"/>
     </item>
     <item row="0" column="3">
      <widget class="QLineEdit" name="leInitials">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="lblCompanyDesc">
       <property name="text">
        <string>Co&amp;mpany:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="buddy">
        <cstring>leCompany</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="lblStreetDesc">
       <property name="text">
        <string>Street:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="buddy">
        <cstring>leStreet</cstring>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="lblPhoneHome">
       <property name="text">
        <string>Home Number:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="buddy">
        <cstring>lePhoneHome</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="leCompany"/>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="lblInitialsDesc">
       <property name="text">
        <string>Initials:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="buddy">
        <cstring>leInitials</cstring>
       </property>
      </widget>
     </item>
     <item row="5" column="3">
      <widget class="QLineEdit" name="lePhoneWork"/>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="lblPositionDesc">
       <property name="text">
        <string>Position:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="buddy">
        <cstring>lePosition</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QLineEdit" name="leCity"/>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="lblEmailDesc">
       <property name="text">
        <string>E&amp;mail:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="buddy">
        <cstring>leEmail</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QLabel" name="lblCityDesc">
       <property name="text">
        <string>Cit&amp;y:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="buddy">
        <cstring>leCity</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="leStreet"/>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="lblFaxDesc">
       <property name="text">
        <string>Fa&amp;x Number:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="buddy">
        <cstring>leFax</cstring>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="leFax"/>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>leFullName</tabstop>
  <tabstop>leInitials</tabstop>
  <tabstop>leTitle</tabstop>
  <tabstop>lePosition</tabstop>
  <tabstop>leCompany</tabstop>
  <tabstop>leEmail</tabstop>
  <tabstop>leStreet</tabstop>
  <tabstop>leCity</tabstop>
  <tabstop>lePostal</tabstop>
  <tabstop>leCountry</tabstop>
  <tabstop>lePhoneHome</tabstop>
  <tabstop>lePhoneWork</tabstop>
  <tabstop>leFax</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
